src/plugins/irc-events/ctcp.js

```diff
@@ -21,14 +21,20 @@ module.exports = function(irc, network) {
 		chan.pushMessage(client, msg);
 	});
 
-	irc.on("ctcp request", function(data) {
+	irc.on("ctcp request", (data) => {
 		switch (data.type) {
-		case "PING":
-			var split = data.message.split(" ");
+		case "PING": {
+			const split = data.message.split(" ");
 			if (split.length === 2) {
 				irc.ctcpResponse(data.nick, "PING", split[1]);
 			}
 			break;
 		}
+		case "SOURCE": {
+			const packageJson = require("../../../package.json");
+			irc.ctcpResponse(data.nick, "SOURCE", packageJson.repository.url);
```

Comment:
So is it expected that this will respond with `https://github.com/thelounge/lounge.git` or should it respond with `https://github.com/thelounge/lounge`?